pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions and manipulate strings.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical or math expression with an operator('+','-','*','&','|' and etc.), special symbols should be escaped:
```

Comment:
You need a space before the "(" and after the commas. Also, "and etc." --> "etc."
Finally, "special symbols should be escaped" must be a separate sentence, for grammatical correctness.